terra/compute/docker.py

```diff
@@ -76,13 +77,24 @@ def run_service(self, service_info):
     else:
       tty_args = ('-T')
 
+    command = service_info.command + extra_arguments
+
+    # If debug_service matches this service name
+    if (debug_service := os.environ.get('TERRA_DEBUG_SERVICE', None)) and \
+       any(
+         [x.__name__ == debug_service for x in service_info.__class__.__mro__]
+       ):
+      print("To start the service runner, run:")
```

Comment:
is this the command the user will run to debug the service?

Reply:
It is the command to execute the service runner that is normally run. They can do whatever they want with it. Options include:

- Ignore it completely
- Run it exactly as is, let an exception happen, and then poke at the files on disk.
- Run it through pdb
- Run python.exe and then run piece manually from there
- Etc...